ts/create-gh-actions/tsconfig.json

```
@@ -0,0 +1,12 @@
+{
+  "compilerOptions": {
+    "target": "es2017",
+    "module": "es2022",
+    "forceConsistentCasingInFileNames": true,
```

`esModuleInterop` してもよいかもしれません ([ref](https://dev.classmethod.jp/articles/esmoduleinterop-flag-for-typescript/))
```suggestion
    "forceConsistentCasingInFileNames": true,
    "esModuleInterop": true,
```